mmaction/models/localizers/bmn.py

```diff
@@ -135,6 +135,8 @@ def __init__(self,
             -0.5, 1.5)
         self.match_map = self._match_map()
         self.bm_mask = self._get_bm_mask()
+        if torch.cuda.is_aviaiable():
+            self.bm_mask = self.bm_mask.cuda()
```

Comment:
Will this work when using multiple gpus? Can it go to the correct device?

Reply:
It can work normally with multiple gpus.